orion-commons/src/main/java/com/pinterest/orion/utils/TeletraanClient.java

```diff
@@ -0,0 +1,280 @@
+package com.pinterest.orion.utils;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.util.EntityUtils;
+
+import java.io.UnsupportedEncodingException;
+import java.net.HttpURLConnection;
+import java.util.logging.Logger;
+
+public class TeletraanClient {
```

Comment:
Should we introduce an interface for instance operations (if there is not one already)? With this interface, we can have Kafka's way of instance operations and Teletraan based operations be implementations of that interface. Then each user will be able to pick which option to choose for which type of system.

Reply:
Make sense. I can add one extra layer called instance helper.